ztag/encoders/protobuf.py

```diff
@@ -63,11 +78,8 @@ def encode(self, zout):
             if valid_nss is not None:
                 c.valid_nss = valid_nss
                 c.validation_timestamp = record.timestamp
-            parents_sha256 = cert_dict.get("parents", None)
-            if parents_sha256 is not None:
-                c.parents.extend([
-                    sha_value.decode("hex") for sha_value in parents_sha256
-                ])
+            c.parents = [p.decode("hex") for p in cert_dict.get("parents", [])]
```

Comment:
I don't think this assignment works because protobuf repeated fields are not real lists.